packages/experimental/Avatar/SPEC.md

```diff
@@ -0,0 +1,311 @@
+# Avatar
+
+## Background
+
+The Avatar component represents a person or entity. It displays the person's image, initials, or an icon, and can be either circular or square.
+Previously it was a component PersonaCoin which used an old version of framework. There are more details in MIGRATION.md.
+
+```ts
+import { JSAvatar } from '@fluentui-react-native/experimental-avatar';
+```
+
+## Sample Code
+
+Basic examples:
+
+```jsx
+  <JSAvatar />
+  <JSAvatar icon={{ svgSource: { uri: 'https://www.example.com/test.svg', viewBox: '0 0 100 100' } }} size={120} />
+  <JSAvatar src="../../path-to-image.png" active="active" activeAppearance="ring" size={56} />
+  <JSAvatar shape="square" avatarColor="purple" name="Richard Feynman" />
+```
+
+More examples on the [Test pages for the Avatar](../../../apps/fluent-tester/src/FluentTester/TestComponents/Avatar). Instructions on running the tester app can be found [here](../../../apps/fluent-tester/README.md).
+
+## Variants
+
+### Active
+
+Optional activity indicator.
+Active prop can be `unset`, `active` and `inactive`. Default is `unset`.
+
+- active: the avatar will be decorated according to activeAppearance, e.g. `ring`.
+- inactive: the avatar will be partially transparent (0.8 opacity).
+- unset: normal display
+
+### ActiveAppearance
+
+According to Web spec there should be `ring`, `shadow`, `glow`, `ring-shadow`, `ring-glow`.
```

Comment:
Link the web spec too perhaps?

Reply:
I think we can just say we support ring. We don't have to compare to the web spec in this file.

Reply:
MIGRATION.md could have that info instead